xml/Microsoft.Extensions.Localization/StringLocalizer`1.xml

```diff
@@ -91,9 +91,9 @@
         <Parameter Name="includeParentCultures" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="includeParentCultures">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="includeParentCultures"><see langword="true" /> to include strings from parent cultures; <see langword="false" /> otherwise.</param>
```

Comment:
This sentence seems a bit off.

Reply:
What would you say feels off about it? It has the usual structure for boolean attributes. Feel free to suggest an alternative, otherwise I would just merge this.

Reply:
How about `Whether to include parent cultures in the search for a resource` similar to what is found in: https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.localization.resourcemanagerstringlocalizer.getallstrings?view=dotnet-plat-ext-3.1

Reply:
From the wiki: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors


| Parameter type | Wording | Examples |
|----------------|---------|----------|
| Boolean | **true** to *XXX*; otherwise, **false**. <br />**true** to *XXX*; **false** to *XXX*.<br /><br /> **Note:**<br />Use the "otherwise" wording unless the second condition must be noted explicitly.<br /><br /> **Note:**<br />The wording for Boolean parameters is "**true** to...," not "**true** if...." | *includePrompt*<br />Type: **System.Boolean**<br />**true** to include prompt characters in the return string; otherwise, **false**. <br /><br />*isChecking*<br />Type: **System.Boolean**<br />**true** to indicate that the object can be frozen (without freezing it); **false** to actually freeze the object. |

Reply:
understood. thanks for explanation.